demo/image-carousel-demo-app.html

```diff
@@ -0,0 +1,29 @@
+<link rel="import" href="../polymer/polymer-element.html">
```

Comment:
This and line 3 will be `../../` instead of `../` for `polymer-cli` to correctly map it.

Reply:
Ok, just updated this.  For my own edification, what goes wrong with the way it was? I'm just using 'polymer build' and 'polymer serve', and it _seemed_ to be working fine.  What am I missing?